jeremy-kendra/README.md

```diff
@@ -0,0 +1,23 @@
+# Project Name
+
+**Author**: Jeremy Beck and Kendra Ellis
+**Version**: 1.1.0 (increment the patch/fix version number up if you make more commits past your first submission)
+
+## Overview
+<!-- Provide a high level overview of what this application is and why you are building it, beyond the fact that it's an assignment for a Code Fellows 301 class. (i.e. What's your problem domain?) -->
+
+## Getting Started
+<!-- What are the steps that a user must take in order to build this app on their own machine and get it running? -->
+
+## Architecture
+HTML, JS, jQuery, ExpressJS, node
+
+## Change Log
+
```

Comment:
you probably want to make this a list so they don't all appear on the same line jumbled together. Make sure you are previewing your markdown files